Billing.API/DopplerSecurity/CryptoHelper.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Security.Cryptography;
+using System.Text;
+using Billing.API.Services.Invoice;
+using Microsoft.Extensions.Options;
+
+namespace Billing.API.DopplerSecurity
+{
+    public class CryptoHelper
+    {
+        private readonly string _signatureHashKey;
+
+        public CryptoHelper(IOptions<InvoiceProviderOptions> options)
+        {
+            _signatureHashKey = options.Value.SignatureHashKey;
+        }
+
+        public string GenerateInvoiceSign(string clientPefix, int clientId, int fileId)
```

Comment:
Que opinás de mi propuesta de hacerlo más generico? 

```suggestion
        public string GenerateInvoiceSign(string payload)
```